library/std/src/os/fuchsia/process.rs

```diff
@@ -0,0 +1,25 @@
+//! Fuchsia-specific extensions to primitives in the [`std::process`] module.
+//!
+//! [`std::process`]: crate::process
+
+use crate::process;
+use crate::sealed::Sealed;
+
+#[unstable(feature = "fuchsia_exit_status_aborted", issue = "none")]
```

Comment:
Maybe just `fuchsia_exit_status`? In case there is eventually more than just `aborted_code`

Reply:
Good point. I changed the feature name to `fuchsia_exit_status`.